src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -330,16 +330,13 @@ export class FormTransferTransactionTs extends FormTransactionBase {
      * @return {TransferTransaction[]}
      */
     protected getTransactions(): TransferTransaction[] {
-        const mosaicsInfo = this.$store.getters['mosaic/mosaics'] as MosaicModel[];
         const mosaics = this.formItems.attachedMosaics
             .filter((attachment) => attachment.uid) // filter out null values
             .map(
                 (attachment): Mosaic => {
                     const amount = Number(attachment.amount.trim().replace(DECIMAL_SEPARATOR, '.'));
-                    const info = mosaicsInfo.find((i) => i.mosaicIdHex === attachment.mosaicHex);
-                    const div = info ? info.divisibility : 0;
-                    // - format amount to absolute
-                    return new Mosaic(new MosaicId(RawUInt64.fromHex(attachment.mosaicHex)), UInt64.fromUint(amount * Math.pow(10, div)));
+                    const mosaicId = new MosaicId(RawUInt64.fromHex(attachment.mosaicHex));
+                    return new Mosaic(mosaicId, UInt64.fromUint(amount));
```

Comment:
this will let transactions have the wrong amount.

Reply:
I resolved this.